tests/functionality_tests/localization_visualizer.py

```diff
@@ -31,9 +31,9 @@ class DataType(IntEnum):
     # Settings, can be changed as desired
     zone = ENGINEERING_QUAD  # Used for GPS visualization
     zone_photo = "geo_images/engineering_quad.png"
-    imu_data_file = "csv/imu_360_sample1.csv"
-    gps_data_file = "csv/GPS_13-11-2021_14-18-15.txt"
-    # gps_data_file = "csv/GPS_22-11-2021.txt"
+    imu_data_file = "./csv_files/imu_360_sample1.csv"
+    # gps_data_file = "./csv_files/GPS_13-11-2021_14-18-15.txt"
+    gps_data_file = "./csv_files/GPS_22-11-2021.txt"
 
     # Read command-line arguments
     if len(sys.argv) != 4 or "live=" not in sys.argv[1] or "data=" not in sys.argv[2] or "ekf=" not in sys.argv[3]\
```

Comment:
how do you pass in the args? I'm confused. Maybe an example in comments?

Reply:
which args? the data files? they're CSVs and we're just reading one line per iteration of forloop 
we're only working with gps/imu data - not with the robot at all (this file goes back to when we didnt even have a robot to work with)

Reply:
like, how would you run this test file?